File: app/src/main/java/co/edu/unipiloto/myapplication/model/RegisterRequest.java
package co.edu.unipiloto.myapplication.model;

public class RegisterRequest {
    public String email;
    public String password;
    public String rol;

    public RegisterRequest(String email, String password, String rol) {
        this.email = email;
        this.password = password;
        this.rol = rol;
    }
}

File: app/src/main/java/co/edu/unipiloto/myapplication/rest/RegisterRequest.kt
package co.edu.unipiloto.myapplication.rest

data class RegisterRequest(
    val fullName: String,
    val email: String,
    val password: String, // Recibimos la contraseña en texto plano (temporalmente)
    val phoneNumber: String?,
    val role: String,
    val sucursal: String?,
    val isActive: Boolean = true
)